platform/ios/MGLPolygon.m

```diff
@@ -0,0 +1,20 @@
+#import "MGLPolygon.h"
+
+@interface MGLMultiPoint (MGLPolygon)
+
+- (instancetype)initWithCoordinates:(CLLocationCoordinate2D *)coords count:(NSUInteger)count;
+- (BOOL)intersectsOverlayBounds:(MGLCoordinateBounds)overlayBounds;
```

Comment:
This is kind of a sneaky way of getting access to these methods. The declarations should go in a private header in platform/ios/.